doc(methods): update wrong imports to update

vit0rr opened the pull request:
Hello :)

These paths to updates are wrong.
To have a reference, follow the [step09](https://github.com/meteor/react-tutorial/blob/master/src/simple-todos/step09/imports/ui/TaskForm.jsx) file. It does not have any `TasksCollection` to update in this file.

The doc: https://react-tutorial.meteor.com/simple-todos/09-methods.html#9-4-api-and-db-folders
![image](https://github.com/meteor/react-tutorial/assets/70543018/2f4547b7-bf86-4303-943e-1a9eae295c1a)

Grubba27 commented:
It should have as stated [here](https://react-tutorial.meteor.com/simple-todos/03-forms-and-events.html#3-4-Add-Submit-Handler)

vit0rr commented:
> It should have as stated [here](https://react-tutorial.meteor.com/simple-todos/03-forms-and-events.html#3-4-Add-Submit-Handler)
> 
> Your file should look like this:
> 
> ```js
> import React, { useState } from 'react';
> import { TasksCollection } from '/imports/api/TasksCollection';
> 
> export const TaskForm = () => {
>   const [text, setText] = useState("");
> 
>   const handleSubmit = e => {
>     e.preventDefault();
> 
>     if (!text) return;
> 
>     TasksCollection.insert({
>       text: text.trim(),
>       createdAt: new Date()
>     });
> 
>     setText("");
>   };
>  
>   return (
>     <form className="task-form" onSubmit={handleSubmit}>
>       <input
>         type="text"
>         placeholder="Type to add new tasks"
>         value={text}
>         onChange={(e) => setText(e.target.value)}
>       />
> 
>       <button type="submit">Add Task</button>
>     </form>
>   );
> };
> ```

You are right. Probably I removed the import at the same time that I removed the `TasksCollection.insert` to use methods. I'll close this PR
